src/style.css

```diff
@@ -0,0 +1,64 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  /* font-weight: 300; */
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.searchbar {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+}
+
+.searchbar.big {
+  height: 70px;
+  padding-left: 62px;
+  background: url(./images/Search.svg) no-repeat 25px 24px / 19px;
+  font-size: 16px;
+}
+
+.searchbar.small {
+  height: 42px;
+  padding-left: 33px;
+  background: url(./images/Search.svg) no-repeat 12px 14px / 11px;
+  font-size: 14px;
+}
+
+.searchbar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchbar:focus {
+  outline: none;
+}
+
+.searchbar.big:focus {
+  background: url(./images/Search.svg) no-repeat 25px 24px / 19px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.searchbar.small:focus {
+  background: url(./images/Search.svg) no-repeat 12px 14px / 11px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.searchbar::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+}
+
+.searchbar:focus::placeholder {
```

Comment:
on focus placeholder do not change at all, only when you write in it, that text should be bold 
but in your case when focus on a particular input box that placeholder's font becomes bold

Reply:
easy peasy ;)